### ISSUE TITLE
[Feature] Add a `comment` template variable

### ISSUE THREAD

**Issue by RyotaUshio** [RyotaUshio/obsidian-pdf-plus#225]
### Describe your request

Currently, we need to write quite a complex javascript to extract annotation comments:
https://github.com/RyotaUshio/obsidian-pdf-plus/discussions/132#discussioncomment-9452904

Solution: provide a simple variable `comment` for this purpose. It can be just an empty string when it's a backlink highlight (i.e. not actual annotation written in the fie)

**Comment by samanthavenialogan** [comment]
Holy crap that acted quickly! Already with the commit! How long do you think it will be before I can update and access the comment variable do you think?

**Comment by RyotaUshio** [comment]
I'll release the update within a few days.

**Comment by RyotaUshio** [comment]
Released in [0.40.8](https://github.com/RyotaUshio/obsidian-pdf-plus/releases/tag/0.40.8).

**Comment by samanthavenialogan** [comment]
Following up after the release of the comments feature. I have replaced the long string you provided to me in case [case #132](https://github.com/RyotaUshio/obsidian-pdf-plus/discussions/132#discussioncomment-9452904) and can confirm that the net 'comment' tag is working precisely as described and does not overwrite annotations created through the 'text' or 'selection' variables. Seems it's working pretty well ^^ 

I greatly appreciate it. 

My current implementation: 
![image](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/54729415/8f17ce5e-1b80-4b1e-9ba8-0b03df7435bf)

```
> [!{{calloutType}} ] {{linkWithDisplay}}
>>**Quote**: _{{selection}}_
>
>>**Comment**:  {{comment}}
```
